systemtest/src/test/java/io/strimzi/systemtest/migration/MigrationST.java

```diff
@@ -654,6 +673,71 @@ private void deleteClusterOperator() {
         LOGGER.info("Cluster Operator Pod deleted");
     }
 
+    /**
+     * Based on {@param withJbodStorage} it returns the {@link Storage} - either with just one Persistent Volume, or with
+     * JBOD storage containing two Persistent Volumes.
+     *
+     * @param withJbodStorage   boolean parameter determining if the JBOD storage should be used in the test case
+     *
+     * @return  {@link Storage} for the particular testcase (single PV/JBOD)
+     */
+    private Storage getStorageBasedOnTestCase(boolean withJbodStorage) {
+        String pvSize = "1Gi";
+
+        PersistentClaimStorage volume = new PersistentClaimStorageBuilder()
+            .withDeleteClaim(true)
+            .withId(METADATA_VOLUME_ID)
+            .withKraftMetadata(KRaftMetadataStorage.SHARED)
```

Comment:
This flag doesn't make sense on brokers when we start from a ZK cluster but luckily it doesn't hurt. I have spoken with Jakub, he will add a warning when it happens but it should have no impact on the STs. Just an heads up for you.

Reply:
I have it there for the Broker pods to have the exact disk where the cluster metadata topic should be present (I thought it will that effect as for the controller Pods).
If it's the case, I don't want to change it during the test, so I just placed it there from the beginning, to make the checks easier.

Reply:
so yes in general, what's going to happen for a user would be ...

1. start from a ZK based cluster where broker node pool doesn't have kraftMedatata flag on JBOD disks
2. migrate to KRaft
3. on brokers the cluster metadata topic will go to the JBOD disk with ID = 0 (or the minimum one)

In your case you are marking the specific disk to be used after the migration for cluster topic on brokers. It's fine anyway.

Reply:
Okay cool, I just wanted to be sure. I guess I can keep it as it is, but thanks for the heads up! :)